config/environments/robots_example.yml

```diff
@@ -43,5 +43,6 @@ sul-robots-host-example:
   - sdr_preservationIngestWF_transfer-object
   - sdr_preservationIngestWF_validate-bag
   - sdr_preservationIngestWF_update-moab
+  - sdr_preservationIngestWF_validate-moab
```

Comment:
do we use robots.yml files anymore?  if not, we should just remove the example file to avoid future confusion

Reply:
Yes, it should be removed.

Reply:
Deleted